test/cli.js

```diff
@@ -31,6 +31,17 @@ test.failing('ignores fixture', async t => {
 	t.throws(execa('../../../cli.js', ['--no-local'], {cwd}));
 });
 
+test('ignore files in .gitignore', async t => {
```

Comment:
To make sure the `cwd` option is adhered to. In the CLI test it just uses `process.cwd()`. API tests are a lot faster to run too. Long-term, I'm going to convert many of the CLI tests to API tests.

I've merged this now, but would be nice if you could submit an API test for this.